site/pages/our-work/case-study/nandos-loyalty/index.js

```diff
@@ -0,0 +1,379 @@
+
+// // Update
+// const social = {
+//   title: "Nando's loyalty case study | Red Badger",
+//   description:
+//     'Red Badger worked with the Nando’s product team to reimagine the customer experience of the restaurant’s loyalty programme. Together we designed and built a digital-first, mobile wallet enabled loyalty experience enabling customers to earn and redeem rewards across all digital touchpoints.',
+//   metaImage: socialImg,
+//   altText: 'TODO',
+//   url: 'https://red-badger.com/our-work/case-study/nandos-loyalty',
+// };
+
+const quoteProps = {
+  text:
+    'None of this would be possible without the audacity and vision to deliver a cutting edge customer experience, willingness to collaborate, adopt a test and learn mindset and staying totally focused on the needs of our customers. Red Badger has been a powerful ally from the start and instrumental in bringing this loyalty programme to market.',
+  author: {
+    name: 'Ryan Foreman',
+    title: "Product Manager at Nando's",
+    image: RyanX2Img,
+  },
+};
+
+// const endQuoteProps = {
+//   text:
+//     "It was a real joy to work with Nando's on a truly innovative, industry first approach to customer loyalty. The brand has a cult following unlike any other among the digital native customers, and the mobile wallet approach clearly fit right in with their lives, allowing more of them to enjoy the Nando's rewards. Overcoming the technical obstacles was great fun for the team and thanks to Nando's openness to innovation, we were able to do some of our best work together.",
+//   author: {
+//     name: 'Viktor Charypar',
+//     title: 'Tech Director at Red Badger',
+//     image: Viktor,
+//   },
+// };
+
+// const tryptychConfig = {
+//   srcImageLeft: left,
+//   srcImageMiddle: middle,
+//   srcImageRight: right,
+//   altImageLeft:
+//     "Phone with the Pride app on in screen displaying a map, text reads: 'Easily find bars and events on the day'",
+//   altImageMiddle:
+//     "Phone with the Pride app on in screen displaying a list of events, text reads: 'Plan your perfect festival with ease and love'",
+//   altImageRight:
+//     "Phone with the Pride app on in screen displaying a list of event categories, text reads: 'Find diverse experiences from cabaret to films'",
+// };
+
+export default function Pride() {
+  return (
+    <ScrollTracker>
+      {
+        // <Social {...social} />
+      }
+      <div className={styles.headerImgWrapper}>
+        <Image
+          src={HeaderX1Img}
+          src2x={HeaderX2Img}
+          src3x={HeaderX3Img}
+          className={styles.headerImg}
+          altText="People (Badgers) sticking post-its on a blue wall"
+        />
+      </div>
+      <Body>
+        <PageHeading>Digital-first loyalty programme</PageHeading>
+        <Section>
+          <SectionBody>
+            <P>
+              <a href="https://www.nandos.co.uk/card/about">Nando&:aposs loyalty programme</a>{' '}
```

Comment:
Also my bad, clearly can't type
```suggestion
              <a href="https://www.nandos.co.uk/card/about">Nando&apos;s loyalty programme</a>{' '}
```

Reply:
Think I fat thumbed a brand new mistake on this one 😅